manifests/enterprise/dashboard/service.pp

```diff
@@ -0,0 +1,38 @@
+# = Class: sensu::enterprise::dashboard
+#
+# Installs the Sensu Enterprise Dashboard
+class sensu::enterprise::dashboard::service (
+  $hasrestart = true,
```

Comment:
Are there some cases when the init/service script doesn't have a restart option?